modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -655,6 +655,11 @@ public ObjectDefinition getObjectDefinition(long objectDefinitionId)
 		return objectDefinitionPersistence.findByPrimaryKey(objectDefinitionId);
 	}
 
+	@Override
+	public List<ObjectDefinition> getObjectDefinitions(long objectFolderId) {
```

Comment:
Could you please run `buildService` here again? I saw a semantic versioning fail that can be related.

Reply:
Ok, thank you